public/js/event.js

```diff
@@ -59,14 +86,216 @@ $(document).ready(function(){
+      eventGoingMaxHTML += "  <div class='cell'>";
+      eventGoingMaxHTML +=     permission;
+      eventGoingMaxHTML += "  </div>";
+      eventGoingMaxHTML += "  <div class='cell'>";
+      eventGoingMaxHTML +=     makeAdminHTML;
+      eventGoingMaxHTML += "  </div>";
+      eventGoingMaxHTML += "  <div class='cell'>";
+      eventGoingMaxHTML +=     chownHTML;
+      eventGoingMaxHTML += "  </div>";
+      eventGoingMaxHTML += "</div>";
+
+      eventGoingMinHTML += "<div class='row'>";
+      eventGoingMinHTML += "  <div class='cell'>";
+      eventGoingMinHTML +=     "<a href=" + "'/profile/" + member + "'>" + name + "</a>";
+      eventGoingMinHTML += "  </div>";
+      eventGoingMinHTML += "  <div class='cell'>";
+      eventGoingMinHTML +=     permission;
+      eventGoingMinHTML += "  </div>";
+      eventGoingMinHTML += "</div>";
+    });
+    return (eventData.owner_id === authenticationData.user_id) ?  $('#eventGoingTable').html(eventGoingMaxHTML) : $('#eventGoingTable').html(eventGoingMinHTML);
+  };
+
+  PrintData.prototype.sectionsAndButtons = function() {
+    if (eventData.like_ids.indexOf(authenticationData.user_id) !== -1) {
+      $('#unlikeEventButton').show();
+    }else {
+      $('#likeEventButton').show();
+    }
+    if (authenticationData.user_id === eventData.owner_id) {
+      $('#editEventButton').show();
+      $('#deleteEventButton').show();
+    }else if (eventData.admin_ids.indexOf(authenticationData.user_id) !== -1) {
+      $('#editEventButton').show();
+      $('#withdrawEventButton').show();
+    }else if (eventData.going_ids.indexOf(authenticationData.user_id) !== -1) {
+      $('#withdrawEventButton').show();
+    }else {
+      $('#joinEventButton').show();
+    }
+  };
+
+  var Post = function() {
+  };
+
+  Post.prototype.joinEvent = function() {
+    $.ajax({
+      context: this,
+      method:   'POST',
+      url:      '/events/' + eventID + '/join',
+      success:  function(reply){
+        console.log(reply);
+        window.location.reload();
+      },
+    });
+  };
+
+  Post.prototype.likeEvent = function() {
+    $.ajax({
+      context: this,
+      method:   'POST',
+      url:      '/events/' + eventID + '/like',
+      success:  function(reply){
+        console.log(reply);
+        window.location.reload();
+      },
+    });
+  };
+
+  Post.prototype.makeAdmin = function(id, payload) {
+    $.ajax({
+      context:  this,
+      method:   'POST',
+      data:     payload,
+      url:      '/events/' + id + '/admin',
+      success:  function(reply){
+        console.log(reply);
+        window.location.reload();
+      },
+    });
+  };
+
+  Post.prototype.chown = function(id, payload) {
+    $.ajax({
+      context:  this,
+      method:   'POST',
+      data:     payload,
+      url:      '/events/' + id + '/chown',
+      success:  function(reply){
+        console.log(reply);
+        window.location.reload();
+      },
+    });
+  };
+
+
+  var Put = function() {
```

Comment:
Why create different classes for `Put` and `Delete`? I think it makes more sense, if anything, to group the by the collections you want to manipulate:

`Event.update()` vs `PUT.updateEvent`